src/service/network.ts

```diff
@@ -131,6 +142,8 @@ export class Wifi extends Service {
 
     get enabled() { return this._client.wireless_enabled; }
     set enabled(v) { this._client.wireless_enabled = v; }
+    get ipv4Address() { return this._device.ip4_config.get_addresses(); }
+    get ipv6Address() { return this._device.ip6_config.get_addresses(); }
```

Comment:
the getter needs to be in snake_case for the gobject getter to work

Reply:
fixed in the commit